language/stdlib/modules/DesignatedDealer.move

```diff
@@ -247,7 +247,7 @@ module DesignatedDealer {
     }
 
     spec module {
-        pragma verify = false;
+        pragma verify = true;
```

Comment:
Aha!  Did you have to make Libra::mint opaque to get the verification to succeed?

Reply:
I did not change anything else.